cloud_functions/strava_auth_initiate/main.py

```
@@ -0,0 +1,165 @@
+except Exception as e:
+    partial_log_structured(
+        message="Error initializing Firebase Admin",
+        error=str(e),
+        level="ERROR",
+        traceback=traceback.format_exc(),
+    )
+    raise
+
+
+@https_fn.on_request(
+    cors=options.CorsOptions(cors_origins=["*"], cors_methods=["POST", "OPTIONS"])
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)